nemo/export/trt_llm/nemo/nemo_ckpt_convert.py

```diff
@@ -122,6 +124,33 @@ def rename_key_dist_ckpt(old_key: str, layer: int):
 
 
 def load_sharded_metadata(checkpoint_dir: Union[Path, TarPath], torch_tensor=True):
+    if isinstance(checkpoint_dir, TarPath):
+        raise NotImplementedError('TarPath not supported for general distributed checkpoints')
```

Comment:
@oyilmaz-nvidia is it ok for now to block the TarPath load?
If not, can you think of someone who could help implement this path?

Reply:
@apanteleev Is this the part you made a change recently?

Reply:
Sorry for a false alarm. I played with TarPath a bit and managed to make it work with the new format.